app/index.html

```diff
@@ -9,16 +9,17 @@
   <meta charset="utf-8">
     <!-- Make the page mobile compatible -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!-- Make the page mobile compatible -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <!-- Allow installing the app to the homescreen -->
     <meta name="mobile-web-app-capable" content="yes" />
-
   <!-- Allow installing the app to the homescreen -->
   <meta name="mobile-web-app-capable" content="yes">
   <link rel="icon" href="/favicon.ico" />
   <script>
     if('caches' in window){
-      caches.keys().then((name)=>{
+      caches.keys().then((names)=>{
         names.forEach(name=>{
```

Comment:
Please follow our workflow of creating all new branches from develop branch because for example, this is not related to this PR's task.